manifests/psp.yaml

```diff
@@ -42,15 +42,16 @@ spec:
     - 'secret'
     - 'downwardAPI'
     - 'hostPath'
+    - 'nfs'
     - 'persistentVolumeClaim'
   allowedHostPaths:
-    - pathPrefix: '/public/dumps'
+    - pathPrefix: '/mnt/public/dumps'
```

Comment:
This is a good point. I think we can drop this entirely as psp is not used in a local dev env. Also this would likely vanish with https://phabricator.wikimedia.org/T317787